code/modules/mob/living/simple_animal/slaughter/bloodnode.dm

```diff
@@ -0,0 +1,120 @@
+//yes i did stealfrom blobcode...
+/obj/effect/bloodnode
+	name = "blood pustilue"
+	icon = 'icons/mob/blob.dmi'
+	icon_state = "blank_blob"
+	var/health = 100
+	anchored = 1
+	density = 1
+	var/lastblood = 0
+	var/nodecount = 0
+	var/datum/artifact_effect/paranoia = new /datum/artifact_effect/badfeeling
+
+
+
+	New(loc, var/h = 100)
```

Comment:
This entire file has a lot of relative pathing, and that's a bad thing.
Each of these procs should have an absolute path and be tabbed appropiately
ex
/obj/effect/bloodnode/New(loc, var/h = 100)
`tab` nodecount += 1